labels-owners.yaml

```diff
@@ -44,8 +49,11 @@ labels:
       - krishnadurai
       - kunmingg
       - lluunn
+      - yanniszark
   area/jupyter:
     owners:
+      - elikatsis
+      - ioandr
```

Comment:
@ioandr and @elikatsis  don't seem that active with respect to Jupyter. Do they really have the context to triage and help out on jupyter related issues?

https://github.com/pulls?q=+is%3Apr+author%3Aioandr+org%3Akubeflow+

https://github.com/pulls?q=is%3Aissue+author%3Aelikatsis+org%3Akubeflow+

Reply:
@ioandr made the original contribution of the Jupyter Web App and @elikatsis has worked on issues we had with the JWA.
My first understanding of label-owners was to add the people who are knowledgeable / own a piece of the code. 
From your comments I understand that it's meant to be used as a list of people active in day-to-day issues, not so much as a record of who knows what.
In that case, I will remove those entries.